README.md

```diff
@@ -1,9 +1,75 @@
 # Docker Containers
-This repository is a collection of a number of Dockerfile configuration files that can be used to achieve different actions surrounding OpenSpace. The files are separated into subfolders that make clear what the files are used for. The rest of this document explains how to install Docker and then how to use each of the folders and what they contain. The general gist is that a Docker container is like a virtual machine, but a lot easier (and faster) to setup. One important difference is that a Docker container will (in general) forget everything that you have done to it in an interactive session. This is great for reproducibility, but can be surprising.
+
+This repository is a collection of a number of Dockerfile configuration files that can be used to achieve different actions surrounding OpenSpace.
+
+The files are separated into subfolders that make clear what the files are used for.
+
+The rest of this document explains how to install Docker and then how to use each of the folders and what they contain. The general gist is that a Docker container is like a virtual machine, but a lot easier (and faster) to setup. One important difference is that a Docker container will (in general) forget everything that you have done to it in an interactive session. This is great for reproducibility, but can be daunting to get on the ground running.
 
 ## Installation
+
+> [!NOTE]
+>
+> Ensure that virtualization is enabled on your computer. Some computers don't have this enabled and require entering the BIOS of the computer.
+>
+> To verify your computer has virtualization enabled:
+>
+> * On Windows (pwsh): `Get-ComputerInfo -property "HyperV*"`
+> * On Linux: `lscpu | grep Virtualization`
+> * On macOS: `sysctl -a | grep -o VMX`
+
 ### Windows
-If you are on Windows 11, the easiest way is to open a Powershell terminal and install Docker from there: `winget install Docker.DockerDesktop`.  Otherwise you can download the installer from [here](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe).
+
+If you are on Windows 11, the easiest way is to open a Powershell terminal and install Docker from there: `winget install Docker.DockerDesktop`. Otherwise you can download the installer from [here](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe).
+
+> [!IMPORTANT]
+>
+> It is best to run the `winget` command in an administrative terminal, to avoid repeated User Account Control (UAC) prompts.
```

Comment:
```suggestion
> [!IMPORTANT]
>
> If you want to avoid repeated User Account Control (UAC) prompts, you can run the `winget` command in an administrative terminal.
```
?

Feeling hesitant to teach people to run things with elevated priviledges